SpinCore_pp/ppg/IR.py

```diff
@@ -126,7 +126,7 @@ def run_IR(
         runBoard()
         run_scans_time_list.append(time.time())
         run_scans_names.append("get data")
-        raw_data = getData(data_length, nPoints, nEchoes, nPhaseSteps)
+        raw_data = getData(data_length, nPoints, nEchoes, nPhaseSteps,'getData needs this str here')
```

Comment:
this is incorrect -- roll back

You need to re-run `python setup.py develop` to resolve the error, rather than doing this.

We edited the c code so that it doesn't take this extra/unused argument.  But, for that to take effect, you need to re-compile.  Recompilation only happens when you run setup.py.